src/common/dom/load_resource.ts

```diff
@@ -23,7 +23,11 @@ const _load = (
         if (type) {
           (element as HTMLScriptElement).type = type;
           // https://github.com/home-assistant/frontend/pull/6328
-          (element as HTMLScriptElement).crossOrigin = "use-credentials";
+          if (isUrlAbsolute(url)) {
+            (element as HTMLScriptElement).crossOrigin = "anonymous";
+          } else {
+            (element as HTMLScriptElement).crossOrigin = "use-credentials";
+          }
```

Comment:
Please use a ternary statement. We also don't need to test if it is isAbsolute, we just need it to be a relative url starting at the root. As in, any url that is `local/module.js` would never work, we only need to catch `/local/module.js` or `/api/hacs/module.js` etc

```suggestion
          (element as HTMLScriptElement).crossOrigin = url.substr(0, 1) === "/" ? "use-credentials" : "anonymous";
```